builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -900,7 +900,7 @@ def writeServerlessFile(config, environment_logical_id, accountDetails){
       sh "sed -i -- '/#Start:isVaultAndVpcEnabled/,/#End:resources/d' ./serverless.yml"    
   } else  {
     sh "sed -i -- '/#Start:resources/,/#End:resources/d' ./serverless.yml"       
-    sh "sed -i -e 's|\${file(deployment-env.yml):iamRoleARN}|${iamRoleArnValue}|g' serverless.yml"
+    sh "sed -i -e 's|\${file(deployment-env.yml):iamRoleARN}|${config['iamRoleARN']}|g' serverless.yml"
```

Comment:
Do all services with `config['domain'] == 'jazz'` have  `iamRoleARN` in their `config`? If not, this might break since the `else` block gets executed when `config['iamRoleARN']` exists OR `config['domain'] != 'jazz'`. Is that expected?

Reply:
All the services with domain == 'jazz' (Core services) will have iamRoleARN in config. And it is expected.